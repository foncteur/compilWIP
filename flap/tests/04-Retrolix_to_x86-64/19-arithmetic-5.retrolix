external printf

def print_one_int ()
local a1:
  a1: a1 <- copy %rdi;
  l10: %rdi <- copy "%d\n";
  l11: %rsi <- copy a1;
  l12: %rax <- copy 0;
  l13: printf();
  l14: ret;
end

globals (x)
  l01: %rdi <- div 37, 73;
  l03: print_one_int();
  l04: exit;
end