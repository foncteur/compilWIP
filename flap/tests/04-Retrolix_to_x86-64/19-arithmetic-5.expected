Process exited with status exited(0).
STDOUT:
0

STDERR:



